client/src/pages/ShoppingLists.js

```diff
@@ -1,13 +1,16 @@
-import React from "react";
+import React, { useState } from "react";
 import { Box } from "@material-ui/core";
 import AddItem from "components/AddItem";
 import ListsDisplay from "components/ListsDisplay";
+import AddList from "components/AddList"
 
 const ShoppingLists = () => {
+    const [addListOpen, setAddListOpen] = useState(false)
```

Comment:
Like we've done elsewhere, let's not pass setAddListOpen directly to the child components; instead add a function here that calls setAddListOpen and pass that function down